plugins/angular.js

```diff
@@ -22,6 +22,10 @@ function ngRavenExceptionHandler(RavenConfig, $delegate) {
     if (!RavenConfig)
         throw new Error('RavenConfig must be set before using this');
 
+    if (RavenConfig.debug !== undefined) {
```

Comment:
Can you change this to `if (RavenConfig.debug != void 0) {`? This matches how we check for "is undefined" elsewhere.